### PR TITLE
Add image to story (backend, fronend get)

### PR DESCRIPTION
**Pull request by matanel-oren**
In order to enable adding image to the stories, I added another db that stores the images.
Point post schema was changed (image_id added to story) but backward compatibility preserves.
Frontend now can fetch also the image if there is one.
Frontend post point progress is on develop by Josh, so I don't touch it. So now you can post an image via backend docs only. This is how you do that: 1. post an image via /image post. 2. take the id that was returned, and insert id in the field "image_id" in /point post. 3. now when the point story is fetched to the fronend the image will be shown within the story.

### CODE REVIEW

**Review by nvgoldin — COMMENTED** [review]
Cool!
Can you add BE tests?

File: backend/zulu/routes.py [hunk 1]
```diff
@@ -34,12 +38,32 @@ def create_point(location: UserLocation, db=Depends(points_db)):
...
+        record = db.find_one({'_id': ObjectId(id)})
+    except InvalidId:
+        raise HTTPException(status_code=404, detail="Image not found. Id is not valid")
+    if not record:
+        raise HTTPException(status_code=404, detail="Image not found")
+    return StreamingResponse(io.BytesIO(record['image']), media_type=record['content_type'])
+
+
+@api.post("/image", status_code=status.HTTP_201_CREATED)
+def add_image(image: UploadFile = File(...), db=Depends(images_db)):
```
Comment (nvgoldin) [inline backend/zulu/routes.py:60]: Cool! Didn't know about `fastapi`'s UploadFIle

File: backend/zulu/routes.py [hunk 2]
```diff
@@ -34,12 +38,32 @@ def create_point(location: UserLocation, db=Depends(points_db)):
...
+    image_id = db.insert({'image': image.file.read(),
+                          'content_type': image.content_type,
+                          'filename': image.filename})
+    return {'id': str(image_id), 'filename': image.filename}
```
Comment (nvgoldin) [inline backend/zulu/routes.py:64]: 1. Also not super-urgent - but can you add a schema for it? should be the ResponseModel
2. I am not sure if we need the filename - I think it depends on the browser implementation what would be the name.
3. One last thing - images should probably be stored as bytes and not string

**Review by nvgoldin — COMMENTED** [review]
Few nits

File: backend/zulu/routes.py (hunk 2 above)
Comment (matanel-oren) [reply backend/zulu/routes.py:64]: 1. Just added.
2. Right. It is only for testing. Can be removed if disturbs.
3. Yes they are. Why do you think they are not?

File: backend/zulu/routes.py (hunk 1 above)
Comment (matanel-oren) [reply backend/zulu/routes.py:60]: Google knows everything..

File: backend/tests/test_routes.py [hunk 3]
```diff
@@ -116,6 +148,83 @@ def test_get_points_two_results(db_data, test_data, client):
...
+
+    near_tel_aviv = [data for data in more_data
+                 if data['user_id'] == 'near Tel Aviv'].pop()
+
+    points = sorted([near_jerusalem, near_tel_aviv], key=lambda x: x['user_id'])
+    images_names = os.listdir(os.path.join(BASE_TESTS_DIR, 'tests_images'))
+    images_types = ['image/' + name[name.rindex('.') + 1:] for name in images_names]
+    for i in range(2):
+        image_path = os.path.join(BASE_TESTS_DIR, 'tests_images', images_names[i])
+        response = client.post("/api/image", files={'image': (images_names[i], open(image_path, 'rb'), images_types[i])})
```
Comment (nvgoldin) [inline backend/tests/test_routes.py:183]: I'd start here by first asserting the response (as it seems it later fails on missing `id` in this response).

**Review by nvgoldin — APPROVED** [review]
Great work :)

**Pull request merged**